### PR TITLE
feat: add Linkedin to the Community Section

### PR DESCRIPTION
**Pull request by annaindistress**
Signed-off-by: Anna Artemova <annartmva@gmail.com>

### Description

- Fix spacing for label in hero section for mobile
- Add LinkedIn link to the Community Section on the Tetragon [homepage](https://deploy-preview-2796--tetragon.netlify.app)

image

image

### CODE REVIEW

**Review by mtardy — COMMENTED** [review]
sure! Thanks, I think you are missing an empty line between your commit title and the signed-off line (see the checkpatch result).

**Review by mtardy — DISMISSED** [review]
Thanks!!! :)

File: docs/content/en/_index.html [hunk 1]
```diff
@@ -11,18 +11,18 @@
                 <span class="hero__label">
                     <span>
                         <span>
-                        Tetragon is a <b>sub-project under</b>
-                    </span>
-                    <img src="/svgs/logos/cillium.svg" width="20" height="20" loading="eager" alt="" />
-                    <span>
+                            Tetragon is a <b>sub-project under</b>
```
Comment (mtardy) [inline docs/content/en/_index.html:14]: ah sorry you have the huge whitespace change here as well. Maybe you want to lint this doc once for all, what would you prefer?

Comment (annaindistress) [reply docs/content/en/_index.html:14]: My bad, I'll fix it right away. I'll just turn off auto linting for this project on my end. 

Also, would it be okay if I fix a few minor spacing issues in the hero section for mobile in this PR as well?

Comment (mtardy) [reply docs/content/en/_index.html:14]: yeah you can write separate commits for that, no worries, or group them, as you wish! :)

**Review by jrfastab — APPROVED** [review]
Thanks!

**Pull request merged**